plugins/scaffolder-backend/src/scaffolder/actions/builtin/fetch/cookiecutter.ts

```diff
@@ -86,12 +100,18 @@ export function createFetchCookiecutterAction(options: {
 
       const cookiecutter = templaters.get('cookiecutter');
 
+      const values = {
+        ...(ctx.input.values as TemplaterValues),
+        _copyWithoutRender: ctx.input.copyWithoutRender,
```

Comment:
I'd also suggest that only setting `_extensions` and `_copy_without_render` if they're actually present and arrays. Otherwise we might end up blowing up cookiecutter if they're undefined in json.

Reply:
JSON serialization will omit `undefined` values, so we're good

Reply:
Yes. I've still added the `Array` check, otherwise Cookiecutter complains,
- `_copy_without_render` is not iterable, and
- looks for an extension called `j` (when asked to use `jinja2_custom_filters_extension`).